### ISSUE TITLE
Harmonize rstudio and docker instructions

### ISSUE THREAD

**Issue by JasonJWilliamsNY** [CyVerse-learning-materials/ez_quickstart#7]
The other two pages of the quickstart should be harmonized with the formatting of the first page, including indentation and use of RST admonitions (code blocks, notes, etc.).

**Comment by bjoyce3** [comment]
Clarification question: there are commands to execute (labeled by double backticks like `` command ``) and there are code blocks that are large chunks of command line output. Are you suggesting to make everything a code block and to not use the ``command`` syntax?

**Comment by JasonJWilliamsNY** [comment]
No, I think you can go ahead and use backticks (this is still true on the first page). However on the RStudio or docker page I remember seeing a block of code, and didn't check (but assumed you hadn't applied the directive - lazy of me). More important I think is just making sure the instructions read in the same way. For instance, all major subheading may need to include an instruction to be logged in. I know it may seem overkill, but making subheadings as self-sufficient as possible means:

1. Someone can link to that subheading and not be lost because the instructions have no context. 
2. When someone searches and find the sat subheading, again, they have enough context to complete instructions in that subheading without reading the whole page. 

Does that make sense? 

Thanks!
